pkg/sfu/downtrack.go

```diff
@@ -316,7 +329,9 @@ func (d *DownTrack) Codec() webrtc.RTPCodecCapability { return d.codec }
 // StreamID is the group this track belongs too. This must be unique
 func (d *DownTrack) StreamID() string { return d.streamID }
 
-func (d *DownTrack) PeerID() livekit.ParticipantID { return d.peerID }
```

Comment:
I thought his API was used in other places, for example StreamAllocator. But, my memory is probably incorrect.